### PR TITLE
feat(packages/sui-pde): add getDecision function

### PR DESCRIPTION
**Pull request by javiauso**
## Description
Add getDecision function to allow features in places like getInitialProps.

image

image

### CODE REVIEW

**Review by jordevo — APPROVED** [review]
nice! 👏

**Review by lolarufino — APPROVED** [review]

**Review by hpintos — APPROVED** [review]

**Review by AgonisticKatai — APPROVED** [review]

File: packages/sui-pde/src/index.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ export {default as PDE} from './pde.js'
 export {default as useFeature} from './hooks/useFeature.js'
 export {default as PdeContext} from './contexts/PdeContext.js'
 export {default as useExperiment} from './hooks/useExperiment.js'
+export {default as getDecision} from './hooks/common/getDecision.js'
```
Comment (kikoruiz) [inline packages/sui-pde/src/index.js:5]: Is it a hook indeed?

Comment (javiauso) [reply packages/sui-pde/src/index.js:5]: Right, I will look for a better location, thanks!

File: packages/sui-pde/src/hooks/common/getDecision.js
[the remark this answers is not in the document]
Comment (javiauso) [reply packages/sui-pde/src/hooks/common/getDecision.js:1]: Agree, I will add it 👌🏼

[the remark this answers is not in the document]
Comment (javiauso) [reply packages/sui-pde/src/hooks/common/getDecision.js:1]: done! @kikoruiz

File: packages/sui-pde/src/index.js (hunk 1 above)
Comment (javiauso) [reply packages/sui-pde/src/index.js]: done! @kikoruiz

**Review by kikoruiz — APPROVED** [review]
❤️

**Review by tomasmax — APPROVED** [review]

**Pull request merged**